engines/query-sparql-link-traversal/package.json

```diff
@@ -49,7 +49,7 @@
     "@comunica/actor-extract-links-predicates": "^0.1.1",
     "@comunica/actor-extract-links-quad-pattern": "^0.1.1",
     "@comunica/actor-extract-links-quad-pattern-query": "^0.1.1",
-    "@comunica/actor-extract-links-tree": "^0.1.1",
+    "@comunica/actor-extract-links-tree": "^0.1.2",
```

Comment:
We shouldn't bump the version manually, as that will mess up our lerna publish command when we do a new release.

Could you revert this change?
Perhaps the CI failure was just a caching issue?